### DISCUSSION TITLE
Sync now button

### DISCUSSION THREAD

**Discussion by mjvrbk** [lavinir/hassio-onedrive-backup#154] [Ideas]
I would like a Sync Now button.
Or is there another way to do this?

**Comment by lavinir** [comment]
Sync happens every 5 minutes. What would be the goal here ?

**Reply by mjvrbk** [reply]
I didn't know that, I was under the assumption that a sync was triggered by a backup.
OneDrive Backup I installed a few days, I'm still discovering.
Made a few backups two days ago, manually.
Deleted one of those online from OneDrive.
The option "Remove deleted files during File Sync is enabled, but the backup remains listed in the interface.
So I thought no sync was taking place.

**Reply by lavinir** [reply]
Sorry that was kind of hard to follow. The "Remove deleted files during File Sync" does not apply to backups but to a different feature for syncing files from HA to OneDrive. The backup deletions occur based on your Backup Retention settings (Max Backups, Generational Settings).

I encourage you to look at the Readme, it likely contains all the answers to your questions.
